plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -185,21 +179,21 @@ def __init__(self, image_shape, height='auto', order=3, twidth=1, is_complex=Fal
                 Ycosn_recon = Ycosn_forward
 
             himask = pointOp(log_rad, self.Yrcos, Xrcos)
-            self._himasks.append(torch.tensor(himask).unsqueeze(0).unsqueeze(-1))
+            self._himasks.append(torch.tensor(himask).unsqueeze(0))
```

Comment:
why have the masks changed shape?

Reply:
because the masks don't need to be unsqueezed for the last dimension since the complex tensor is now just 4 dimensional instead of 5